tracer/test/Directory.Build.props

```
@@ -20,7 +20,7 @@
   </ItemGroup>
 
   <ItemGroup Condition=" $(DD_LOGGER_ENABLED) != 'false' ">
```

question: should we change this to 

```suggestion
  <ItemGroup Condition=" $(DD_LOGGER_ENABLED) == 'true' ">
```

So that the logger requires explicit enabling, rather than explicit disabling? I'm undecided, as it makes default local different to default to CI, but also feels "correct" in that we don't add the reference unless we explicitly ask for it